Awards update 23

kylemtam opened the pull request:
I added the CANRGX award from 2019 as well as the 2021 awards. I didn't add anything from 2023 as I don't have the photo of the trophy while we don't know if we'll get anything from Launch Canada.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/waterloo-rocketry/website-react/30)
<!-- Reviewable:end -->

QuantumManiac reviewed (changes requested):
> Reviewed 4 of 4 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/website-react/30)* status: all files reviewed, 4 unresolved discussions (waiting on @kylemtam, @lucasreljic, and @ryan-lam)

___
*[`src/components/AwardItem/AwardItem.module.css` line 2 at r1](https://reviewable.io/reviews/waterloo-rocketry/website-react/30#-NdQ1RUsEwTTZp_8qGiN:-NdQ1RUsEwTTZp_8qGiO:b-fgqb4e) ([raw file](https://github.com/waterloo-rocketry/website-react/blob/fa81900b1c0f7aa8c81dd51ceebd2e55705e78ef/src/components/AwardItem/AwardItem.module.css#L2)):*
> ```CSS
> .award__wrapper {
> 	width: 40rem;
> ```

Ooh, good catch!
___
*[`src/img/awards/CANRGXOutreach19.png` line 0 at r1](https://reviewable.io/reviews/waterloo-rocketry/website-react/30#-NdQ1VH5E-kiB9UVMyrH:-NdQ1VH5E-kiB9UVMyrI:b-oujuvo) ([raw file](https://github.com/waterloo-rocketry/website-react/blob/fa81900b1c0f7aa8c81dd51ceebd2e55705e78ef/src/img/awards/CANRGXOutreach19.png#L0)):*
Could we get the two pictures down in size a bit more? 2MB each is pretty large. We can probably get even more size savings by just reducing the resolution of the images.

tbh we should probably take all of our pics and make them `jpg`s (or even `webp`s!). Probably something for the future though.
___
*[`src/routes/Awards.jsx` line 12 at r1](https://reviewable.io/reviews/waterloo-rocketry/website-react/30#-NdQ1iyv8LR9Vc8kb1T5:-NdQ1iyv8LR9Vc8kb1T6:b-205j04) ([raw file](https://github.com/waterloo-rocketry/website-react/blob/fa81900b1c0f7aa8c81dd51ceebd2e55705e78ef/src/routes/Awards.jsx#L12)):*
> ```JavaScript
> import SRAD18Image from '../img/awards/10k_SRAD18.png';
> import SRAD17Image from '../img/awards/10k_SRAD17.png';
> import TechincalExcellenceImage from '../img/awards/technical_excellence.png';
> ```

Would you mind sneaking in a fix for this typo (`TechincalExcellenceImage` -> `TechnicalExcellenceImage`)? :pleading_face: 
___
*[`src/routes/Awards.jsx` line 23 at r1](https://reviewable.io/reviews/waterloo-rocketry/website-react/30#-NdQ2P3A738qcc77VV37:-NdQ2P3A738qcc77VV38:b-l619t8) ([raw file](https://github.com/waterloo-rocketry/website-react/blob/fa81900b1c0f7aa8c81dd51ceebd2e55705e78ef/src/routes/Awards.jsx#L23)):*
> ```JavaScript
>           <AwardItem
>             RocketName="KotS"
>             AwardName="DR. GIL MOORE AWARD FOR INNOVATION"
> ```

We should probably just put a `text-transform:  uppercase` in for this. Do it yourself if you'd like, otherwise I can put an issue up for this.


<!-- Sent from Reviewable.io -->

kylemtam commented:
I think I've addressed all of your changes Chamath.

As per the jpg and webp thing, changing the images to jpg might be interesting as they all have transparent backgrounds rn and jpg doesn't support that last time I checked. Not sure about webp but some photo software sometimes have trouble dealing with webp (not sure tho why people are downloading our trophy images lol)

Please note I uh also added the IREC 2023 trophy cuz Artem blessed me with a photo last minute. We are still missing the LC trophy but I think I'll just open an issue for that. Another thing I'd love is if we could support non-trophy awards. E.g. I believe in 2021 we got a few recognitions like RUNNER-UP FOR JIM FURFARO AWARD FOR TECHNICAL EXCELLENCE and TOP 10 FINALIST - SDL PAYLOAD CHALLENGE. I wasn't what the best way to set this up in react (whether there's a way to not include certain divs in the AwardItem element or whethre it'd be easier to just make a new thing).
